scripts/start-rc.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
```

Comment:
why?

Reply:
`function ...` syntax only works in Bash, we have to remove `function` to make it compatible with the Bourne shell. I didn't check for other Bash-isms in the script, there could be more.